itou/approvals/notifications.py

```diff
@@ -0,0 +1,23 @@
+from itou.utils.emails import get_email_message
+from itou.utils.notifications.base_class import NotificationBase
+
+
+class NewProlongationToAuthorizedPrescriberNotification(NotificationBase):
+    """
+    Notification sent to the authorized prescriber supposed to have validated the prolongation.
+    """
+
+    def __init__(self, prolongation):
+        self.prolongation = prolongation
+
+    @property
+    def email(self):
+        to = [self.prolongation.validated_by.email]
+        context = {"prolongation": self.prolongation}
+        subject = "approvals/email/new_prolongation_for_prescriber_subject.txt"
+        body = "approvals/email/new_prolongation_for_prescriber_body.txt"
+        return get_email_message(to, context, subject, body)
+
+    @property
+    def name(self):
+        return "confirm_prolongation_email"
```

Comment:
Je suis très étonnée que ça passe car il s'agit de l'ancien système. 
Dans le nouveau, mis en prod lundi, [le nom de la notification est dans une variable de classe](https://github.com/betagouv/itou/blob/master/itou/utils/notifications/base_class.py#L23] et non plus dans une propriété.  Je suppose donc que ta branche n'intègre pas mes derniers changements.
Idéalement, je conseille plutôt de surcharger la méthode `recipients_emails` en utilisant [`get_recipients()`](https://github.com/betagouv/itou/blob/master/itou/utils/notifications/base_class.py#L48) ([comme ici](https://github.com/betagouv/itou/blob/master/itou/job_applications/notifications.py#L25)) au lieu de renseigner le `to`. Mais dans ton cas ce n'est pas nécessaire car la personne ne peut pas choisir de se désabonner.

C'est super en tous cas de commencer à l'utiliser dans un autre cadre qu'un abonnement / désabonnement ! On pourrait progressivement remplacer les e-mails par ce système. Cela permettrait de mieux les retrouver et de les faire évoluer plus facilement, par exemple si un jour on décide d'intégrer un autre système comme les SMS.

Reply:
Oui, j'avais du code ancien, je vais intégrer le nouveau.

Reply:
Rien que pour voir rapidement quelles notifs on envoie, ça vaut le coup de les déplacer je trouve.